app/main/views/services.py

```diff
@@ -305,15 +308,15 @@ def delete_draft_service(framework_slug, lot_slug, service_id):
                                 delete_requested=True))
 
 
-@main.route('/frameworks/<framework_slug>/submissions/documents/<int:supplier_id>/<document_name>', methods=['GET'])
+@main.route('/frameworks/submissions/<framework_slug>/submissions/<int:supplier_id>/<document_name>', methods=['GET'])
```

Comment:
I think I got really confused with all the URLs in that Slack discussion. Looking at it now, I think we don't need the `submissions` part after `/frameworks/` since our common URL prefix for the views is `/frameworks/<framework_slug>/submissions`. So the URL could be just `/frameworks/<framework_slug>/submissions/<int:supplier_id>/<document_name>`. Which makes the URL nicer without the duplicate `submissions` bit, but I think it might create some possible ordering issues with `/suppliers/frameworks/<framework_slug>/submissions/<lot_slug>` views.


We could change it to something like `/documents/<framework_slug>/submissions/<int:supplier_id>/<document_name>` and move it away from framework URLs altogether instead.